ts/screens/wallet/payment/TransactionErrorScreen.tsx

```diff
@@ -166,6 +187,7 @@ export const errorTransactionUIElements = (
         requestAssistanceForPaymentFailure(rptId, paymentHistory);
         break;
       case ToolEnum.zendesk:
+        requestZendeskAssistanceForPaymentFailure(rptId, paymentHistory);
         handleZendeskRequestAssistance?.();
```

Comment:
we don't need to handle this has a function possibly `undefined`, since it is not.
I suggest to change the `errorTransactionUIElements` signature

from
```typescript
export const errorTransactionUIElements = (
  maybeError: NavigationParams["error"],
  rptId: RptId,
  onCancel: () => void,
  choosenTool: ToolEnum,
  paymentHistory?: PaymentHistory,
  handleZendeskRequestAssistance?: () => void
):
```

to
```typescript
const errorTransactionUIElements = (
  maybeError: NavigationParams["error"],
  rptId: RptId,
  onCancel: () => void,
  choosenTool: ToolEnum,
  handleZendeskRequestAssistance: () => void,
paymentHistory?: PaymentHistory
):
```

Reply:
Modified in [ca56d27](https://github.com/pagopa/io-app/pull/3593/commits/ca56d27cb2d4509aba347a8967be8ecb2e152eb3)